Dockerfile.in

```diff
@@ -1,29 +1,41 @@
 FROM DISTRO
+
 # Docker build arguments
 ARG SOURCE_DIR=/ffmpeg
 ARG ARTIFACT_DIR=/dist
+
 # Docker run environment
-ENV DEB_BUILD_OPTIONS=noddebs
-ENV DEBIAN_FRONTEND=noninteractive
-ENV ARCH=BUILD_ARCHITECTURE
-ENV GCC_VER=GCC_RELEASE_VERSION
-ENV LLVM_VER=LLVM_RELEASE_VERSION
-ENV SOURCE_DIR=/ffmpeg
-ENV ARTIFACT_DIR=/dist
-ENV TARGET_DIR=/usr/lib/jellyfin-ffmpeg
-ENV DPKG_INSTALL_LIST=${SOURCE_DIR}/debian/jellyfin-ffmpeg6.install
-ENV PATH=${TARGET_DIR}/bin:${PATH}
-ENV PKG_CONFIG_PATH=${TARGET_DIR}/lib/pkgconfig:${PKG_CONFIG_PATH}
-ENV LD_LIBRARY_PATH=${TARGET_DIR}/lib:${TARGET_DIR}/lib/mfx:${TARGET_DIR}/lib/xorg:${LD_LIBRARY_PATH}
-ENV LDFLAGS="-Wl,-rpath=${TARGET_DIR}/lib -L${TARGET_DIR}/lib"
-ENV CXXFLAGS="-I${TARGET_DIR}/include $CXXFLAGS"
-ENV CPPFLAGS="-I${TARGET_DIR}/include $CPPFLAGS"
-ENV CFLAGS="-I${TARGET_DIR}/include $CFLAGS"
+ENV DEBIAN_FRONTEND=noninteractive \
+    SOURCE_DIR=/ffmpeg \
+    ARTIFACT_DIR=/dist \
+    DEB_BUILD_OPTIONS=noddebs \
+    ARCH=BUILD_ARCHITECTURE \
+    GCC_VER=GCC_RELEASE_VERSION \
+    LLVM_VER=LLVM_RELEASE_VERSION \
+    TARGET_DIR=/usr/lib/jellyfin-ffmpeg \
+    DPKG_INSTALL_LIST=${SOURCE_DIR}/debian/jellyfin-ffmpeg6.install \
+    PATH=${TARGET_DIR}/bin:${PATH} \
+    PKG_CONFIG_PATH=${TARGET_DIR}/lib/pkgconfig:${PKG_CONFIG_PATH} \
+    CFLAGS="-I${TARGET_DIR}/include $CFLAGS" \
+    CPPFLAGS="-I${TARGET_DIR}/include $CPPFLAGS" \
+    CXXFLAGS="-I${TARGET_DIR}/include $CXXFLAGS" \
+    LDFLAGS="-Wl,-rpath=${TARGET_DIR}/lib -L${TARGET_DIR}/lib" \
+    LD_LIBRARY_PATH=${TARGET_DIR}/lib:${TARGET_DIR}/lib/mfx:${TARGET_DIR}/lib/xorg:${LD_LIBRARY_PATH}
 
 # Prepare Debian build environment
 RUN apt-get update \
  && apt-get upgrade -y \
- && yes | apt-get install -y apt-transport-https curl ninja-build debhelper gnupg wget devscripts mmv equivs git nasm pkg-config subversion dh-autoreconf libpciaccess-dev libwayland-dev libx11-dev libx11-xcb-dev libxcb-dri2-0-dev libxcb-dri3-dev libxcb-present-dev libxcb-shm0-dev libxcb-sync-dev libxshmfence-dev libxext-dev libxfixes-dev libxcb1-dev libxrandr-dev libzstd-dev libelf-dev python3-pip zip unzip tar flex bison
+ && apt-get install --no-install-recommends -y \
```

Comment:
`apt-transport-https` is removed?

Reply:
Yes we don't need it anymore on new version of Ubuntu and Debian, but maybe it can be a problem because of the previous one cause the build is using different versions for it so maybe put it back would be a good idea, and from the log I can see error on apt update

Reply:
Well it's better to keep it as this build script needs to support multiple Debian/Ubuntu versions.

Reply:
Yes of course would change this